CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Unreleased
+----------
+
+* Allow configuring sibling module independence in layer contracts.
```

Comment:
Maybe better: "Add support for non-independent sibling modules in layer contracts."